packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/ScatterPlotViewer/components/ScatterPlot/components/Selections/Selections.js

```diff
@@ -231,6 +236,9 @@ function Selections({
       className='brushLayer'
       focusable='true'
       onKeyDown={onKeyDown}
+      onPointerDown={e => cancelEvent(e)}
+      onPointerMove={e => cancelEvent(e)}
+      onPointerUp={e => cancelEvent(e)}
```

Comment:
Cancel selection events so they don't bubble up and trigger dragging on the data plot.